assets/queries/ansible/aws/cloudformation_in_use/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "cloudformation_in_use",
+  "queryName": "AWS CloudFormation In Use",
```

Comment:
Change query name to "AWS CloudFormation Without Template", to be more specific and better represent the vulnerability. Do the same in the corresponding Terraform query, please.